### PR TITLE
[ADD] cooperator_website_uppercase_lastname

### PR DESCRIPTION
**Pull request by robinkeunen**
Install this module to restore current behavior after #301 is merged.

### CODE REVIEW

**Comment by robinkeunen** [comment]
@enricostano

**Review by enricostano — APPROVED** [review]

File: cooperator_website_uppercase_lastname/controllers/main.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    @http.route(
+        ["/subscription/subscribe_share"],
+        type="http",
+        auth="public",
+        website=True,
+    )
+    def share_subscription(self, **kwargs):
+        if "lastname" in kwargs:
+            kwargs["lastname"] = kwargs["lastname"].upper()
+        return super(WebsiteSubscriptionUppercase, self).share_subscription(**kwargs)
```
Comment (huguesdk) [inline cooperator_website_uppercase_lastname/controllers/main.py]: this is python 3, so `super()` without arguments is enough.

**Review by huguesdk — APPROVED** [review]
clean and effective! 👍

File: cooperator_website_uppercase_lastname/controllers/main.py (hunk 1 above)
Comment (robinkeunen) [reply cooperator_website_uppercase_lastname/controllers/main.py]: 👍

**Comment by robinkeunen** [comment]
thx :-)

**Pull request merged**